nnfabrik/examples/mnist/trainer.py

```diff
@@ -7,8 +7,10 @@
 import torch.optim as optim
```

Comment:
prefer you do `from torch import optim`. `as ...` should really be used only when you intend to rename it.

Reply:
Agreed. Weird that pycharm automatically imports it like that though.